libs/ui/src/components/NeoModalHead/NeoModalHead.scss

```diff
@@ -9,22 +9,26 @@
   @include touch {
     & {
       padding: 1.5rem 1.25rem;
-      @include mobile {
-        display: flex;
-        align-items: center;
-        padding: 0 1.25rem !important;
-        min-height: $navbar-mobile-min-height !important;
-      }
-      @include ktheme {
-        background: theme('background-color-inverse');
-      }
+
       .modal-card-title {
         font-size: 1rem;
         line-height: 1.5;
-        @include ktheme {
+      }
+
+      @include ktheme {
+        background: theme('background-color-inverse');
+
+        .modal-card-close-button, .modal-card-title {
           color: theme('text-color-inverse');
         }
       }
+
+      @include mobile {
+        display: flex;
+        align-items: center;
+        padding: 0 1.25rem !important;
+        min-height: $navbar-mobile-min-height !important;
+      }
```

Comment:
why is it inside `@include touch`?

Reply:
I do not know as it was there before, but I can move it out.